python/graphframes/graphframe.py

```diff
@@ -225,6 +225,84 @@ def degrees(self) -> DataFrame:
             .agg(F.count("*").alias("degree"))
         )
 
+    def typeOutDegree(self, edgeTypeCol: str) -> DataFrame:
```

Comment:
I'm trying to slowly move python client to a "pythonic" style (#713), so, can I ask to rename it to `type_out_degree(self, edge_type_col)` (and other methods too)?